install/ansible/tasks/bootstrap_centos.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- name: "Install required packages on CentOS "
+  yum:
+    name: "{{ item }}"
+    state: latest
+    with_items:
+      - unzip
+      - tar
+      - btrfs-progs-devel
+      - device-mapper-devel
+      - libseccomp-devel
+      - util-linux 
+      - socat
+      - libselinux-python
```

Comment:
Why do we need this?

Reply:
http://docs.ansible.com/ansible/latest/intro_installation.html#managed-node-requirements